tools/build_config/Jenkinsfile

```diff
@@ -225,16 +250,12 @@ pipeline {
               module load cmake/\$CMAKE_VERSION &&
               module load matlab/\$MATLAB_VERSION &&
               module load gcc/\$GCC_VERSION &&
-              ./tools/build_config/build.sh --build \
-                  --cmake_flags "-DHerbert_RELEASE_TYPE=$RELEASE_TYPE" \
-                  --matlab_release \$MATLAB_VERSION
+              ./tools/build_config/build.sh --build
```

Comment:
Be useful to know where the herbert release type and matlab version are now set.

Reply:
Do you mean the CMake arg or the environment variable? The CMake is in the configure stage as one might hope with usual CMake setups. The environment var is unchanged.